File: UI/StoryBoard/src/app/dtos/user-dto/user-dto.ts
export class UserDto {
    id : number = 0;
    name : string = "";
    login : string = "";
    password : string = "";
    active : boolean = true;
    adm : boolean = true;

    static createFromFormValues(formValues: any): UserDto {
        const user = new UserDto();
        user.name = formValues.name;
        user.login = formValues.login;
        user.password = formValues.password;
        user.active = true;
        user.adm = false;
        return user;
    }
}

File: WebAPI2/StoryBoard/Dto/UserDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dto;

public class UserDto
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Login { get; set; }
    public string Password { get; set; }
    public bool Active { get; set; }
    public bool Adm { get; set; }
}

